tests/phpunit/src/Commands/Ide/IdeHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
The refactoring of IdeRequiredTestTrait is necessary because trait methods cannot be referenced outside of a class in PHP 8.1.